app/src/main/java/connected_coop/projectapex/PageAdapter.kt

```diff
@@ -0,0 +1,24 @@
+package connected_coop.projectapex
+
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentStatePagerAdapter
+
+class PageAdapter(
+        fragmentManager: FragmentManager,
+        private val numTabs: Int
+): FragmentStatePagerAdapter(fragmentManager) {
+
+    override fun getItem(position: Int): Fragment {
+        return when (position) {
+            0 ->  RecordingPageFragment()
+            1 ->  ResultActivity()
```

Comment:
Sorry, why is 1 tab leading to a fragment and the other to an activity ,jjust seems weird.

Reply:
I'll change it so results activity is a fragment as well!